### DISCUSSION TITLE
I would like to know where I went wrong with my Markdown syntax.

### DISCUSSION THREAD

**Discussion by sisyphusla** [kwchang0831/svelte-QWER#83] [Q&A] [answered]
I attempted to write a blog post, and everything was going smoothly until I tried to use the Footnote syntax. 
I made sure to follow the official `package.json` and even removed all the `node_modules` before running `pnpm i`. 
I referred to the official documentation and the `/user/blogs/example/index.md` for guidance, but I still couldn't get the Footnote syntax to work. 
Strangely, it previews fine in VS Code's `Markdown Preview Enhanced`, which leaves me puzzled as to why it doesn't work in the browser.

Below are my browser details:

`
Google Chrome 118.0.5993.117
MacBook Air M1
macOS Version: 12.5
`
![demo](https://github.com/kwchang0831/svelte-QWER/assets/98180611/c1b87f90-13ed-4552-b142-ac26da5a9b70)

**Comment by kwchang0831** [comment]
Okay... without changing the code on how it analyzes the markdown syntax...
Try giving an extra space between footnote references.

eg.
```md

HaHa[^1][^2]Haha

Haha[^3]

[^1]: This is 1st note.

[^2]: This is 2nd note.

[^3]: This is 3rd note.

// ^--- extra space
```

This should output the result that you are looking for.

**Reply by toddLiao469469** [reply, marked as answer]
I think the root cause of this issue is the behavior of `marked` (markdown processor)

For instance, if you write a footnote with just one space like this:

```md
text [^1]

[^1]: text
```
`marked` will process `[^1]` as a link.

However, if you use two or more spaces:

```md
text [^1]

[^1]: text text
//--------^----
```
`marked` will process `[^1]` as a paragraph. So QWER will process it into a footnote successfully.
